docs/security/03-07-production-profile.md

```diff
@@ -0,0 +1,102 @@
+   - The Hydra OAuth2 server and the Hydra Maester controller have Istio sidecars disabled, destinationRule custom resources are created for these components.
+   - A job that reads the generated database credentials and saves them to the configuration of Hydra is added.
+
+### Persistence
+
+The production profile for the OAuth2 server enables persistence and creates a database in which Hydra saves the registered OAuth2 clients. When you configure Hydra to use the production profile, a PostgreSQL database is installed
+using the [official charts](https://github.com/helm/charts/tree/master/stable/postgresql).
+The database is created in the cluster as a StatefulSet and uses a PersistentVolume that is provider-specific. This means that the PersistentVolume used by the database uses the default StorageClass of the cluster's host provider.
+
+Alternatively, you can use a compatible external database to store the registered client data. To use an external database, you must create a Kubernetes Secret with the database password in the same Namespace as the Hydra OAuth2 server. The details of the external database are passed through these parameters of the production profile override:
+
+| Parameter |  Description |
+|----------|------|
+| **data.postgresql.enabled** | Defines if Hydra should initiate the deployment of and use an in-cluster database. Set to `false` to use an external database. If set to `true`, Hydra always uses an in-cluster database and ignores the external database details. |
+| **data.global.ory.hydra.persitance.user** | Specifies the name of the user with permissions to access the database. |
+| **data.global.ory.hydra.persitance.secretName** | Specifies the name of the Secret in the same Namespace as Hydra that stores the database password. |
+| **data.global.ory.hydra.persitance.secretKey** | Specifies the name of the key in the Secret that contains the database password. |
+| **data.global.ory.hydra.persitance.dbUrl** | Specifies the database URL. For more information, read [this](https://github.com/ory/hydra/blob/master/docs/config.yaml) document. |
+| **data.global.ory.hydra.persitance.dbName** | Specifies the name of the database saved in Hydra. |
+| **data.global.ory.hydra.persitance.dbType** | Specifies the type of the external database. Supported protocols are `postgres`, `mysql`, `cockroach`. Follow [this](https://github.com/ory/hydra/blob/master/docs/config.yaml) link for more information. |
+
+## Use the production profile
+
+You can deploy a Kyma cluster with the Hydra OAuth2 server configured to use the production profile, or you can configure Hydra in a running cluster to use the production profile. Follow these steps:
+
+<div tabs>
+  <details>
+  <summary>
+  Install Kyma with production-ready Hydra
+  </summary>
+  >**NOTE:** Using this configuration installs a PorstgreSQL database in the Kyma cluster.
```

Comment:
The note is [not properly rendered](https://github.com/kyma-project/kyma/blob/da4196b05eb521f9994567c06af7359188d3b4ad/docs/security/03-07-production-profile.md#use-the-production-profile)